### DISCUSSION TITLE
Error in getting started:  ZlibError: zlib: unexpected end of file

### DISCUSSION THREAD

**Discussion by ezaz-ahmed** [honojs/hono#2083] [Q&A]
I wanted to try hono for the first time. 
Whenever I try to getting started, I get some problems. 

```
C:\Users\Ezaz\AppData\Local\npm-cache\_npx\6bbb1d6f54609fb5\node_modules\create-hono\bin:1113
            this[_onError](new ZlibError(err));
                           ^

ZlibError: zlib: unexpected end of file
    at Unzip.write (C:\Users\Ezaz\AppData\Local\npm-cache\_npx\6bbb1d6f54609fb5\node_modules\create-hono\bin:1113:28)
----
----
----
    at ReadStream.end (C:\Users\Ezaz\AppData\Local\npm-cache\_npx\6bbb1d6f54609fb5\node_modules\create-hono\bin:649:31) {
  code: 'Z_BUF_ERROR',
  errno: -5,
  recoverable: false,
  file: 'C:\\Users\\Ezaz\\.degit\\github\\honojs\\starter/2d868772b0b21b5db79a6d65c8769b1c4bf6cdf2.tar.gz',
  cwd: 'D:\\Work\\pro-tutor',
  tarCode: 'TAR_ABORT'
}
```

I tried install it without node v18.19.0 and node v20.11.0. 
Also I tried to use template cloudflare-workers and nodejs.

**Comment by yusukebe** [comment]
Hi @ezaz-ahmed 

It seems that `create-hono` does not work well in your environment. We should make `create-hono` work in different environments, but this is not possible right now. As a work-around, `create-hono` copies from the following starter template, so you can look at the repository directly.

https://github.com/honojs/starter/tree/main/templates

**Comment by alifyasa** [comment]
I ran into the same issue, but got it fixed by deleting the `~/.degit/github/honojs/` folder and rerunning `npm create hono@latest`.
